Tasks/Kulik/task4/task4.py

```diff
@@ -0,0 +1,29 @@
+while True:
+    str_length = (input('Please, enter the length of line which is more than 35 elements: '))
+    if not str_length.isdigit():
+        print('Error! Please, enter the digits.')
+        continue
+    str_length = int(str_length)
+    if str_length <= 35:
+        print('Error! Please, enter the length of line which is more than 35 elements.')
+        continue
+    break      
+with open('text.txt', 'r', encoding = 'utf-8') as f:
+  with open('changed_text.txt', 'w', encoding = 'utf-8') as f_new:
+        text = f.read().split() 
```

Comment:
not the most optimal solution, you read the whole file in memory but it stays open till the end of your program, -1 point for the unwise decision